hexl/ntt/inv-ntt-avx512.hpp

```diff
@@ -14,15 +14,15 @@ namespace hexl {
 /// @param[in, out] operand Input data. Overwritten with NTT output
 /// @param[in] n Size of the transfrom, i.e. the polynomial degree. Must be a
 /// power of two.
-/// @param[in] modulus Prime modulus. Must satisfy q == 1 mod 2n
-/// @param[in] root_of_unity_powers Powers of inverse 2n'th root of unity in
+/// @param[in] modulus Prime modulus q. Must satisfy q == 1 mod 2n
+/// @param[in] inv_root_of_unity_powers Powers of inverse 2n'th root of unity in
```

Comment:
Doxygen would usually catch this, except we only require docstrings for public header files, i.e. those in `hexl/include`.